sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/percentiles.scala

```diff
@@ -390,10 +390,6 @@ case class PercentileCont(left: Expression, right: Expression, reverse: Boolean
   override protected def withNewChildrenInternal(
       newLeft: Expression, newRight: Expression): PercentileCont =
     this.copy(left = newLeft, right = newRight)
-
-  override def orderingFilled: Boolean = left != UnresolvedWithinGroup
```

Comment:
This is always false now?

Reply:
Yes.

Reply:
Why do we make such a change?

Reply:
Before the PR https://github.com/apache/spark/pull/48748, the default value is false.
That is a change not necessary!

Reply:
Because the left should be always `UnresolvedWithinGroup` with `PercentileContBuilder`.

Reply:
I think the previous version is better as we don't need this hidden assumption. I also don't think this PR improves the code, as requiring the implementation to implement these method can force them to think about the semantic.

Reply:
Sounds good to me. Let me restore them.